ansible/library/interface_facts.py

```diff
@@ -21,6 +21,44 @@
 from ansible.module_utils.basic import *
 from collections import defaultdict
 
+DOCUMENTATION = '''
+---
+module: interfaces_facts
+version_added: "1.1"
+author: John Arnold
+short_description: Retrive interface facts from device
+description:
+    - Retrieve interface facts for a device, the facts will be
+      inserted to the ansible_facts key.
+options:
+    all_ports:
+        description:
+            - all ports that expected to be at link up status
+            - this parameter doens't limit how many ports can
+              be present on the device, interface_facts returns
+              facts of all existing ports.
+        required: false
+    except_port:
```

Comment:
>except_port [](start = 4, length = 11)

ignore_ports

Reply:
This parameter has been removed.

Thanks,
Ying